hack/build/print-workspace-status.sh

```diff
@@ -45,6 +45,8 @@ STABLE_IMAGE_REPOSITORY ${DOCKER_IMAGE_REPOSITORY:-cockroach-operator}
 IMAGE_REGISTRY ${DEV_REGISTRY:-us.gcr.io/chris-love-operator-playground}
 
 CLUSTER ${K8S_CLUSTER:-gke_chris-love-operator-playground_us-central1-a_test}
+GIT_REV_LIST $(git rev-list $(git rev-parse --abbrev-ref HEAD) | wc -l)
```

Comment:
How about we name this to describe its contents. Like one of:

```suggestion
NUMBER_COMMITS_ON_BRANCH $(git rev-list $(git rev-parse --abbrev-ref HEAD) | wc -l)
```

```suggestion
GIT_REV_LIST_COUNT $(git rev-list $(git rev-parse --abbrev-ref HEAD) | wc -l)
```

Reply:
@chrisseto or @jlinder I stink at names.  Which one do you want?

Reply:
`NUMBER_COMMITS_ON_BRANCH` is actually descriptive, let's go with that?

Reply:
> `NUMBER_COMMITS_ON_BRANCH` is actually descriptive, let's go with that?

I prefer that one as well.